tensorflow_datasets/image/caltech_birds_test.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function 
+from tensorflow_datasets.image import caltech_birds
+import tensorflow_datasets.testing as tfds_test
+
+
+class CaltechBirdsTest(tfds_test.DatasetBuilderTestCase):
+
+    DATASET_CLASS = caltech_birds.CaltechBirds2010
+
+    SPLITS = {  # No. of train and test samples
+      'train': 9,
+      'test': 6,
+    }
+
+    DL_EXTRACT_RESULT = ['Lists.tgz', 'Annotations.tgz', 'Images.tar.gz']
+
+
+class CaltechBirdsTest(tfds_test.DatasetBuilderTestCase):
```

Comment:
I think you forgot that? You have 2 test class. Delete it the above is better.

Reply:
Yes, I have 2 test classes for two different datasets (CaltechBirds2010 and CaltechBirds2011) similar to what has been done in cifar10 and cifar100.
Could please clarify on what fake examples I have to delete.

Reply:
Sorry I miss it 2011 but if datasets file structure are same. You can inherit like `mnist_test`

Reply:
Unfortunately the datasets do not have the same structure.